internal/build/cmd/generate/commands/gensource/command.go

```diff
@@ -1,304 +0,0 @@
-// SPDX-License-Identifier: Apache-2.0
-//
-// The OpenSearch Contributors require contributions made to
-// this file be licensed under the Apache-2.0 license or a
-// compatible open source license.
-//
-// Modifications Copyright OpenSearch Contributors. See
-// GitHub history for details.
-
-// Licensed to Elasticsearch B.V. under one or more contributor
-// license agreements. See the NOTICE file distributed with
-// this work for additional information regarding copyright
-// ownership. Elasticsearch B.V. licenses this file to you under
-// the Apache License, Version 2.0 (the "License"); you may
-// not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//    http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing,
-// software distributed under the License is distributed on an
-// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-// KIND, either express or implied.  See the License for the
-// specific language governing permissions and limitations
-// under the License.
-
-package gensource
```

Comment:
I think we should keep the generation code since a lot of it can be reused for #284.

Reply:
The generation code would need a complete rewrite as I change the structure of the lib. Moreover we would need to exclude it from linting and testing as it would brake the current workflows.

The code is still present in all the 2.x branches, so if someone wants to re use it they can get it from the branches and push it back to main once updated.